languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -9026,6 +9026,17 @@ though, and has since been slightly modified)-->
         <example correction="trois fois par semaine">Je joue au Softball <marker>trois fois au semaine</marker>.</example>
       </rule>
       <!-- End of additions -->
+      <rule id="VOUS_LE_PPA_1ER_GROUPE" name="s manquant à vous le ppa" default="temp_off">
+          <pattern>
+              <token>vous</token>
+              <token>le</token>
```

Comment:
Would this also work with more freedom in the second token?

_C'est comme vous le dite._
_Pourquoi vous faite cela ?_
_Vous ne me dite jamais rien._

By the way, we will have to prioritize this rule over `CONFUSION_RULE_PREMIUM` since that rule would suggest the infinitive here. Otherwise it won't even show up in the diff...